test/fixtures/xml/zeebe-taskListeners.bpmn

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0qh9wc7" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.23.0" modeler:userTaskPlatform="Camunda Cloud" modeler:userTaskPlatformVersion="8.0.0">
+  <bpmn:process id="ZeebePropertiesTest" name="Zeebe Properties Test" isExecutable="true">
+    <bpmn:extensionElements>
+      <zeebe:taskListeners>
+        <zeebe:taskListener eventType="complete" retries="1" type="complete_listener" />
+      </zeebe:taskListeners>
+    </bpmn:extensionElements>
+    <bpmn:userTask id="Activity_0ls77qd">
+      <bpmn:extensionElements>
+        <zeebe:taskListeners>
+          <zeebe:taskListener eventType="assign" retries="2" type="assign_listener" />
+        </zeebe:taskListeners>
+      </bpmn:extensionElements>
+    </bpmn:userTask>
+    <bpmn:subProcess id="Activity_1yf35jc">
+      <bpmn:extensionElements>
+        <zeebe:taskListeners>
+          <zeebe:taskListener eventType="complete" retries="3" type="complete_listener" />
+        </zeebe:taskListeners>
+      </bpmn:extensionElements>
+      <bpmn:userTask id="Event_06ifkc6">
+        <bpmn:extensionElements>
+          <zeebe:taskListeners>
+            <zeebe:taskListener eventType="assign" retries="4" type="assign_listener" />
+          </zeebe:taskListeners>
+        </bpmn:extensionElements>
+      </bpmn:userTask>
+    </bpmn:subProcess>
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="ZeebePropertiesTest">
+      <bpmndi:BPMNShape id="Activity_0ls77qd_di" bpmnElement="Activity_0ls77qd">
+        <dc:Bounds x="160" y="400" width="100" height="80" />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Activity_1yf35jc_di" bpmnElement="Activity_1yf35jc" isExpanded="true">
+        <dc:Bounds x="160" y="510" width="350" height="200" />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Event_06ifkc6_di" bpmnElement="Event_06ifkc6">
+        <dc:Bounds x="200" y="592" width="36" height="36" />
+      </bpmndi:BPMNShape>
```

Comment:
Is this a copy paste result? The user task looks funny ;)

![image](https://github.com/user-attachments/assets/9eda50fd-d9ce-435d-8ca4-0d9ad4d494a1)

Reply:
```suggestion
      <bpmndi:BPMNShape id="NestedUserTask_di" bpmnElement="NestedUserTask">
        <dc:Bounds x="200" y="592" width="100" height="80" />
      </bpmndi:BPMNShape>
```

Reply:
yes, this is copy paste result. Should I change it or it doesn't matter?

Reply:
I left the suggestion so that you can apply it from the UI. This one is not blocking.